docs/en/sql-reference/statements/select/except.md

```diff
@@ -146,8 +146,73 @@ Result:
 └─────────────┘
 ```
 
+# EXCEPT Column(s)
 
-**See Also**
+`EXCEPT` can also be used to remove unwanted columns, especially used with a sub query (e.g. `row_number()`).
+
+## Example
+
+The example starts with a table with 2 columns:
+
+```sql
+CREATE TABLE test1  (
+    name String,
+    created_at Datetime
+) ENGINE=MergeTree ORDER BY (name);
+```
+
+Prepare some data:
+
+```sql
+INSERT INTO test1 VALUES ('Felix', NOW()), ('Bob', NOW());
+```
+
+To show the basic usage, you can select all fields except `created_at`:
+
+```sql
+SELECT * EXCEPT(created_at) FROM test1
+```
+
+Cases happen that we have duplicated entries:
+
+```sql
+INSERT INTO test1 VALUES ('Felix', NOW()), ('Bob', NOW());
+```
+
+A simple `select` query would show all versions:
+```text
+┌─name──┬──────────created_at─┐
+│ Bob   │ 2024-07-03 15:06:38 │
+│ Felix │ 2024-07-03 15:06:38 │
+└───────┴─────────────────────┘
+┌─name──┬──────────created_at─┐
```

Comment:
Why do the values in the first and second part have the same timestamps in `created_at`? Are they not created at different times?

Reply:
Yeah, they are different, `38`s vs `35` for the seconds, please take a look again?

Reply:
But it's not so obvious. I'll update it for better illustration

Reply:
Done.